roles/kube_prometheus/vars/main.yml

```diff
@@ -0,0 +1,7 @@
+kube_prometheus_dashboards:
+  - kube-prometheus-calico2-dashboard
+  - kube-prometheus-calico-dashboard
+  - kube-prometheus-elasticsearch-dashboard
+  - kube-prometheus-etcd-dashboard
+  #- kube-prometheus-node-exporter-dashboard
+  - kube-prometheus-prometheus-dashboard
```

Comment:
If the `nginx-ingress` dashboard is not listed here, it will not be deployed :)

Reply:
Yes. We also might want to put this in `defaults/main.yml` to be overriden if needed (a dashboard that get problem to be deployed for example...)